service/collector/chart_operator.go

```diff
@@ -56,3 +83,23 @@ func (c *ChartOperator) Describe(ch chan<- *prometheus.Desc) error {
 	// TODO
 	return nil
 }
+
+func (c *ChartOperator) collectForTenantCluster(ch chan<- prometheus.Metric, cluster tenantCluster) error {
+	ctx := context.Background()
+
+	helmClient, err := c.tenant.NewHelmClient(ctx, cluster.id, cluster.apiDomain)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	_, err = helmClient.GetReleaseHistory(ctx, key.ChartOperatorReleaseName())
```

Comment:
Should emitted metric include `chart-operator` Helm release status information? [Status](https://github.com/giantswarm/helmclient/blob/master/types.go#L16) from `ReleaseContent`?